packages/protocol/contracts/common/GasPriceMinimum.sol

```diff
@@ -153,14 +151,17 @@ contract GasPriceMinimum is Ownable, Initializable, UsingRegistry {
     view
     returns (uint256)
   {
-    FractionUtil.Fraction memory one = FractionUtil.Fraction(1, 1);
-    FractionUtil.Fraction memory blockDensity = FractionUtil.Fraction(
-      blockGasTotal,
-      blockGasLimit
+    int256 blockDensity = toFixed(blockGasTotal).divide(
```

Comment:
Maybe we should stop using `uint256`. Since we have 1 bit less since it's an `int` and then 20 decimals places. Don't know what the size of it, but we can probably use a smaller `uint`

Reply:
I mean, we can use types on our favor...